client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -113,23 +166,23 @@
     @Option(names = "--external-authentication-redirect-handler", paramLabel = "<externalAuthenticationRedirectHandler>", description = "External authentication redirect handlers: ${COMPLETION-CANDIDATES} " + DEFAULT_VALUE, defaultValue = "ALL")
     public List<ExternalRedirectStrategy> externalAuthenticationRedirectHandler = new ArrayList<>();
 
-    @Option(names = "--source", paramLabel = "<source>", defaultValue = "trino-cli", description = "Name of source making query " + DEFAULT_VALUE)
```

Comment:
I should revert removing the defaultValue here, as I later figured out how to identify which options are set by the user.